akka-actor/src/main/resources/reference.conf

```diff
@@ -109,6 +109,10 @@ akka {
     # this is only intended for testing.
     serialize-messages = off
 
+    # Additional serialization bindings which are enabled automatically when allow-java-serialization is disabled.
+    # settings are provided
+    java-serialization-disabled-additional-serialization-bindings = {} ${?akka.actor.java-serialization-disabled-additional-serialization-bindings}
```

Comment:
why `${?`

this is the same thing as the ordinary `serialization-bindings` and we don't have this complicated stuff there